rest_framework/fields.py

```diff
@@ -903,7 +909,12 @@ def enforce_timezone(self, value):
         When `self.default_timezone` is not `None`, always return aware datetimes.
         """
         if (self.default_timezone is not None) and not timezone.is_aware(value):
-            return timezone.make_aware(value, self.default_timezone)
+            # If a timezone is active, we want to use it, but if not we want to use the timezone
+            # specified for this field over the system's default timezone.
+            if hasattr(timezone._active, 'value'):
```

Comment:
There aren't. This is the way the timezone library does it internally so it's fairly safe, at least for now. Maybe I should make a pr to django for a has_active_timezone() method.